changelog.json

```
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "1.6.2-beta2": [
+      "[Added] Add manual conflict resolution options in merge conflicts modal - #6062",
```

Tentative feels on this - in the `1.5.0` changelog we didn't say whether it was a dialog or a modal. If have settled on a name for this somewhere else let me know.

```suggestion
      "[Added] Add manual conflict resolution options in merge conflicts dialog - #6062",
```

---

Thanks for the suggestion @shiftkey! I think I'm going to make a slightly different suggestion that doesn't specify the place.